### PR TITLE
[DDIDO-277]: Add a trigger to build dockerhub images.

### PR DESCRIPTION
**Pull request by nicksantamaria**
Adds a trigger to rebuild the edge images in dockerhub on a weekly schedule.

## Closing issues

DDIDO-277

(originally created by @steveworley on July 10 2020)

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -24,3 +24,34 @@ jobs:
       - run:
           name: Test images
           command: tests/run.sh
+
+  trigger_dockerhub:
+    docker:
+      - image: integratedexperts/ci-builder
```
Comment (nicksantamaria) [inline .circleci/config.yml]: Can we change this to `singledigital/bay-ci-builder:edge` ? @steveworley

File: .circleci/scripts/trigger-dockerhub.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+#
+# Trigger a dockerhub build.
+#
+
+for ENDPOINT in "${!DOCKERHUB_TRIGGER_@}"; do
```
Comment (nicksantamaria) [inline .circleci/scripts/trigger-dockerhub.sh:7]: For anyone else watching, this is looping through these variables
Screen Shot 2020-07-16 at 1 40 55 pm

File: .circleci/config.yml (hunk 1 above)
Comment (steveworley) [reply .circleci/config.yml]: Yeah - I was actually thinking about doing that too after I noticed that we had one set up - will fix it up.

**Review by nicksantamaria — COMMENTED** [review]
seal-of-approval

**Review by steveworley — APPROVED** [review]

**Pull request merged**